packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -509,32 +509,26 @@ describe('<Tabs />', () => {
       expect(container.querySelectorAll(selector)).to.have.lengthOf(1);
     });
 
-    it('should response to scroll events', function test() {
+    it('should response to scroll events', async function test() {
```

Comment:
We should remove this test, since now it does not respond to scroll events (`onScroll` is removed).